x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -515,7 +515,9 @@ static RoleDescriptor kibanaSystem(String name) {
                         "logs-tenable_io.vulnerability-*",
                         "logs-rapid7_insightvm.vulnerability-*",
                         "logs-rapid7_insightvm.asset_vulnerability-*",
-                        "logs-carbon_black_cloud.asset_vulnerability_summary-*"
+                        "logs-carbon_black_cloud.asset_vulnerability_summary-*",
+                        "logs-extrahop.investigation-*",
+                        "logs-qualys_gav.asset-*"
```

Comment:
@ShourieG, can create a new list as this list for 3rd party integrations required for CDR workflow.

Say: https://github.com/elastic/elasticsearch/commit/245d69d9e6cc1340fdd4544ab909d06d4d11ddc6 and with comments to your usecase.